pennylane/ops/channel.py

```diff
@@ -58,6 +58,10 @@ class AmplitudeDamping(Channel):
     @classmethod
     def _kraus_matrices(cls, *params):
         gamma = params[0]
+
+        if gamma < 0.0 or gamma > 1.0:
```

Comment:
Alternatively, you could have
```suggestion
        if not 0 <= gamma <= 1.0:
```

I don't think this is any more efficient, but it _might_ be more readable?

Reply:
Sure, it looks more readable to me.